File: WowUp/WowUp/Models/AddonFolder.cs
namespace WowUp.Models
{
    public class AddonFolder
    {
        public string Name { get; set; }
        public string Path { get; set; }
        public string Status { get; set; }
        public string ThumbnailUrl { get; set; }
        public string LatestVersion { get; set; }

        public Toc Toc { get; set; }
    }
}

File: wowup-electron/src/app/models/wowup/addon-folder.ts
import { FsStats } from "../../../common/models/ipc-events";
import { Addon } from "../../../common/entities/addon";
import { Toc } from "./toc";
import { CurseFolderScanResult } from "../../../common/curse/curse-folder-scan-result";
import { AppWowUpScanResult } from "./app-wowup-scan-result";

export interface AddonFolder {
  name: string;
  path: string;
  status: string;
  ignoreReason?: AddonIgnoreReason;
  thumbnailUrl?: string;
  latestVersion?: string;
  tocs: Toc[];
  matchingAddon?: Addon;
  fileStats?: FsStats;
  cfScanResults?: CurseFolderScanResult;
  wowUpScanResults?: AppWowUpScanResult;
}
